policykit/integrations/slack/models.py

```diff
@@ -228,7 +228,7 @@ def join_conversation(self, channel, users):
 class SlackPostMessage(GovernableAction):
     ACTION = "chat.postMessage"
     AUTH = "admin_bot"
-    EXECUTE_PARAMETERS = ["text", "channel", "thread"]
+    EXECUTE_PARAMETERS = ["text", "channel"]
```

Comment:
why did we remove this?

Reply:
I recalled that I made this change because of a bug I found in the deployment (though forgot exactly what the bug was). For reference, this variable is only used in old policykit logic when the user wants to propose a new action directly on the policykit website. However, the SlackPostMessage does not have a thread field. In the no-code interface, users could still have the thread field because another variable 'EXECUTE_VARIABLES' right after this line determines this.